shared/src/packages/stripe/libs/validation-schemas/get-payments-business.validation-schema.ts

```diff
@@ -0,0 +1,9 @@
+import Joi from 'joi';
```

Comment:
```suggestion
import joi from 'joi';
```
Just like in the previous file. I believe it's the same in other validation schemas.